src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -958,7 +972,11 @@ void GcsActorManager::LoadInitialData(const EmptyCallback &done) {
                    << ", and the number of created actors is " << created_actors_.size();
     for (auto &item : registered_actors_) {
       auto &actor = item.second;
-      if (actor->GetState() != ray::rpc::ActorTableData::ALIVE) {
+      if (actor->GetState() != ray::rpc::ActorTableData::ALIVE &&
+          actor->GetState() != ray::rpc::ActorTableData::DEPENDENCIES_UNREADY) {
```

Comment:
Can you change this to `PENDING_CREATION or RESTARTING"? That will be more explicit.

Reply:
fixed